diff --git a/src/frontend/startkde/plasma-login-kwin_wayland.service.in b/src/frontend/startkde/plasma-login-kwin_wayland.service.in
index d7c7861..9e697e7 100644
--- a/src/frontend/startkde/plasma-login-kwin_wayland.service.in
+++ b/src/frontend/startkde/plasma-login-kwin_wayland.service.in
@@ -3,6 +3,6 @@ Description=KDE Window Manager (Login Manager Version)
 PartOf=graphical-session.target
 
 [Service]
-ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/kwin_wayland --no-lockscreen --no-global-shortcuts --no-kactivities
+ExecStart=@kwin_wayland@ --no-lockscreen --no-global-shortcuts --no-kactivities
 BusName=org.kde.KWin
 Slice=session.slice
