diff --git a/app/models/global_setting.rb b/app/models/global_setting.rb
index 6e06e180..c6c9f807 100644
--- a/app/models/global_setting.rb
+++ b/app/models/global_setting.rb
@@ -256,6 +256,7 @@ class GlobalSetting
         user_name: GlobalSetting.smtp_user_name,
         password: GlobalSetting.smtp_password,
         enable_starttls_auto: GlobalSetting.smtp_enable_start_tls,
+        ca_file: "/etc/ssl/certs/ca-certificates.crt",
         open_timeout: GlobalSetting.smtp_open_timeout,
         read_timeout: GlobalSetting.smtp_read_timeout,
       }
