diff --git a/lib/discourse_js_processor.rb b/lib/discourse_js_processor.rb
index 4a30220b7c..fd88e83a92 100644
--- a/lib/discourse_js_processor.rb
+++ b/lib/discourse_js_processor.rb
@@ -54,7 +54,7 @@ class DiscourseJsProcessor
   end
 
   class Transpiler
-    TRANSPILER_PATH = "tmp/theme-transpiler.js"
+    TRANSPILER_PATH = "app/assets/javascripts/theme-transpiler.js"
 
     @mutex = Mutex.new
     @ctx_init = Mutex.new
@@ -70,7 +70,7 @@ class DiscourseJsProcessor
     end
 
     def self.build_production_theme_transpiler
-      File.write(TRANSPILER_PATH, build_theme_transpiler)
+      File.write(TRANSPILER_PATH, build_theme_transpiler) if !File.file?(TRANSPILER_PATH)
       TRANSPILER_PATH
     end
 
